tests/supervisor/test_execution.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+import logging
+
+import pytest
+from aleph_message.models import ItemHash
+
+from aleph.vm.conf import settings
+from aleph.vm.controllers.firecracker import AlephFirecrackerProgram
+from aleph.vm.models import VmExecution
+from aleph.vm.orchestrator import metrics
+from aleph.vm.storage import get_message
+
+
+@pytest.mark.asyncio
+async def test_create_execution():
```

Comment:
Add a warning that this will make change to the local system (like in TESTING.md)
Maybe also add a mark to no make execution optional since it need to be run as root?

Reply:
Should the tests use temporary directories for the storage paths and set them in the settings as to not make change to the local system ?

Regarding tests running as root, while this test may not need it, I am afraid that many tests will require it for network configuration.